src/Component/Cart/Cart.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Cart.scss";
+import EmptyTable from "./Component//EmptyTable/EmptyTable";
+import ProductTable from "./Component/ProductTable/ProductTable";
+import FillAsidebar from "./Component/Asidebar/FillAsidebar";
+import EmptyAsidebar from "./Component/Asidebar/EmptyAsidebar";
+// import Navbar from "../../Component/Navbar/NavbarStore/"
+import Footer from "./Component/Footer/Footer";
+
+// const API =
+class Cart extends Component {
+  constructor() {
+    super();
+    this.state = {
+      cartList: [],
+      isChecked: true,
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/cartItem.json", {
+      method: "GET",
+    })
```

Comment:
- get method는 default 삭제할 수 있습니다. 
- localhost는 npm start 횟수에 따라 포트 번호가 달라질 수 있는데 포트 번호가 달라지면 fetch가 되지 않기 때문에 삭제해주시면 됩니다. 
- api 주소는 config.js 에서 변수에 선언해 관리해주시면 api주소가 바뀔 때 한번에 관리할 수 있어 편리합니다! 

```suggestion
    fetch("/data/cartItem.json")
```
```suggestion
    fetch(${cartItemAPI})
```

Reply:
오 이렇게도 할 수 있군요! 감사합니다 ㅎㅎㅎ